qiita_db/metadata_template/prep_template.py

```diff
@@ -871,7 +881,7 @@ def _get_predecessors(workflow, node):
                          'artifact transformation']
         merging_schemes = {
             qdb.archive.Archive.get_merging_scheme_from_job(j): {
-                x: y.id for x, y in j.outputs.items()}
+                x: str(y.id) for x, y in j.outputs.items()}
```

Comment:
The conversion to string for y.id is also not obvious here (see below). Perhaps a comment as to why it's necessary would be appropriate?